pkg/ttnpb/rights.go

```diff
@@ -65,6 +65,7 @@ var (
 	AllClusterRights      = &Rights{}
 	AllAdminRights        = &Rights{}
 	AllRights             = &Rights{}
+	AllCollaboratorRights = &Rights{}
```

Comment:
Considering that this is only used in a single place, I don't think we should add this here.